src/main/java/randoop/generation/GrtImpurityAndNumStatements.java

```diff
@@ -0,0 +1,31 @@
+package randoop.generation;
+
+import randoop.sequence.Sequence;
+
+/**
+ * Represents the result of fuzzing a sequence of statements, and the number of additional
+ * statements that were used to fuzz the inputs.
+ */
+class GrtImpurityAndNumStatements {
+
+  /** The sequence that create the inputs. */
+  public Sequence sequence;
+
+  /**
+   * The number of additional statements that were executed to fuzz the inputs. These fuzzing
+   * statements are the last `numStatements` statements in the sequence. If `numStatements` is 0,
```

Comment:
Good comment.
Nit: each occurrence of `` `numStatements` `` should be `{@code numStatements}`.